packages/assets/src/detections/parsers/detectAvif.ts

```diff
@@ -9,13 +10,10 @@ export const detectAvif: FormatDetectionParser = {
     },
     test: async (): Promise<boolean> =>
     {
-        if (!globalThis.createImageBitmap) return false;
-
         // eslint-disable-next-line max-len
         const avifData = 'data:image/avif;base64,AAAAIGZ0eXBhdmlmAAAAAGF2aWZtaWYxbWlhZk1BMUIAAADybWV0YQAAAAAAAAAoaGRscgAAAAAAAAAAcGljdAAAAAAAAAAAAAAAAGxpYmF2aWYAAAAADnBpdG0AAAAAAAEAAAAeaWxvYwAAAABEAAABAAEAAAABAAABGgAAAB0AAAAoaWluZgAAAAAAAQAAABppbmZlAgAAAAABAABhdjAxQ29sb3IAAAAAamlwcnAAAABLaXBjbwAAABRpc3BlAAAAAAAAAAIAAAACAAAAEHBpeGkAAAAAAwgICAAAAAxhdjFDgQ0MAAAAABNjb2xybmNseAACAAIAAYAAAAAXaXBtYQAAAAAAAAABAAEEAQKDBAAAACVtZGF0EgAKCBgANogQEAwgMg8f8D///8WfhwB8+ErK42A=';
-        const blob = await settings.ADAPTER.fetch(avifData).then((r) => r.blob());
 
-        return createImageBitmap(blob).then(() => true, () => false);
+        return testImageFormat(avifData);
```

Comment:
Can't we simply do `new Image` here instead of blob/createImageBitmap? This PR seems like a lot of work to see if AVIF works. We are not trying to test if blob + createImageBitmap is supported only if the format is supported by the browser.

Reply:
We can't use `Image`, because it isn't available in workers.